src/main/java/subSystems/Arm.java

```diff
@@ -0,0 +1,74 @@
+package subSystems;
+
+import com.flash3388.flashlib.robot.RunningRobot;
+import com.flash3388.flashlib.scheduling.Subsystem;
+import com.flash3388.flashlib.time.Time;
+import com.revrobotics.CANSparkMax;
+import com.flash3388.flashlib.robot.control.PidController;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class Arm extends Subsystem {
+
+    private double angle2Target;
+    private CANSparkMax master;
+    private CANSparkMax follower;
+    private DutyCycleEncoder encoder;
+    private PidController pid;
+    private static final double KP = 0;
+    private static final double KI = 0;
+    private static final double KD = 0;
+    private static final double KF = 0;
+    private static final double ERROR = 0;
```

Comment:
Set error and limit values- it's an angle, so don't use set too much but not so little as well. Choose from 0.5-1